velox/type/Timestamp.h

```diff
@@ -340,7 +353,11 @@ struct Timestamp {
         epochToUtc(seconds_, tm),
         "Can't convert seconds to time: {}",
         seconds_);
-    return tmToString(tm, nanos_, options);
+    char* chars = new char[getCapacity(options)];
```

Comment:
auto capacity = getCapacity(options);
char chars[capacity]
Then we don't need to delete.